dockerfiles/init/modules/che-kubernetes-helm/templates/deployment.yaml

```diff
@@ -0,0 +1,209 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: che
+  name: che
+spec:
+  replicas: 1
+  revisionHistoryLimit: 2
+  selector:
+    matchLabels:
+      app: che
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      annotations:
+        pod.alpha.kubernetes.io/init-containers: '[{"image":"busybox","imagePullPolicy":"IfNotPresent","name":"fmp-volume-permission","command":["chmod","777","/data"],"volumeMounts":[{"mountPath":"/data","name":"che-data-volume"}]}]'
+      labels:
+        app: che
+    spec:
+      containers:
+      - env:
+        - name: CHE_DOMAIN
```

Comment:
Is it used by the Che Server?

Reply:
It is in prep for host-based routing in workspaces